quickstarts/mongodb-prometheus/config.yml

```diff
@@ -0,0 +1,67 @@
+id:
+
+# Name of the quickstart
+slug: mongodb-prometheus
+
+# Title of the quickstart
+title: MongoDB (Prometheus)
+
+# Description of the quickstart
+description: |
+  ## MongoDB Monitoring
+
+  MongoDB enables the unlimited virtual scaling of applications. Utilities like mongostat and mongotop offer immediate results but fail to provide insights into trends in a highly graphical visual dashboard. MongoDB dashboards provide insights into key metrics like RAM usage, operations per second, page fault, disk size, lock %, and app and database performance at a glance. 
+
+  Avoid slow queries with  proper indexes that impact performance. Instantly monitor your entire MongoDB database with our instant observability kit or the MongoDB free monitoring tool.  
+
+  The critical differences between MongoDB free monitoring and monitoring MongoDB with New Relic’s instant observability quickstart are efficiency, usability, scope, and cost. MongoDB free monitoring focuses on standalone instances and replica sets. Data collected on disk utilization, memory, and operation execution times are uploaded periodically. 
+
+
+  ### What’s Included?
+
+  New Relic + MongoDB quickstart - New Relic’s instant observability quickstart provides multiple monitoring parameters like operations per second, transactions, and queries by default.
```

Comment:
... quickstart provides multiple metrics related to instances, replica sets, members, databases, and collections.

Reply:
👍